WordPress/Classes/ViewRelated/Views/List/WPStyleGuide+List.swift

```diff
@@ -0,0 +1,13 @@
+/// Convenience constants catalog related for styling in snippet list.
```

Comment:
Should we change "snippet" in this comment?

Reply:
Yes, thank you. 😄 Fixed on the latest commit.